docs/census-open-data-registry.yaml

```diff
@@ -0,0 +1,37 @@
+Name: CZ CELLxGENE Discover Census
+Description: The Census provides efficient computational tooling to access, query, and analyze all single-cell RNA data from CZ CELLxGENE Discover.
+Documentation: https://chanzuckerberg.github.io/cellxgene-census/
+Contact: cellxgene@chanzuckerberg.com
+ManagedBy: "[Chan Zuckerberg Initiative](http://www.chanzuckerberg.com/)"
+UpdateFrequency: New releases are published weekly. LTS releases are published every 6 months.
+Tags:
+  - single-cell
+  - genomics
+  - computational biology
+License: MIT License
+Resources:
+  - Description: CELLxGENE Census Data
+    ARN: arn:aws:s3:::cellxgene-census/cell-census
+    Region: us-west-2
+    Type: S3 Bucket
+DataAtWork:
+  Tutorials:
+    - Title: Computing on the cell by gene matrix using online (incremental) algorithms
```

Comment:
nit: The title on the URL below is `Computing on X using online (incremental) algorithms`

Reply:
I changed the title since I don't think X is particularly clear in this context. This page is meant for people who don't know what the dataset is about. That said, we can change it to something better.

Reply:
Ah, gotcha

Reply:
We should probably get @pablo-gar to weigh in.

Reply:
re-write looks good